src/OpenTelemetry.Instrumentation.GrpcNetClient/Implementation/GrpcClientDiagnosticListener.cs

```diff
@@ -88,8 +88,9 @@ public override void OnStartActivity(Activity activity, object payload)
                 // propagator is used, as HttpClient by itself will only
                 // do TraceContext propagation.
                 var textMapPropagator = Propagators.DefaultTextMapPropagator;
+                var context = new PropagationContext(activity.Context, Baggage.Current);
```

Comment:
Did you intend to use `CreateFromActivity` here?

Reply:
Missed that thanks. Updated